com.unity.render-pipelines.universal/Editor/Shadow/ShadowCascadeSplitGUI.cs

```diff
@@ -167,7 +177,6 @@ public static void HandleCascadeSliderGUI(ref float[] normalizedCascadePartition
 #else
                                 s_OldSceneLightingMode = s_RestoreSceneView.m_SceneLighting;
 #endif
-                                s_RestoreSceneView.cameraMode = SceneView.GetBuiltinCameraMode(DrawCameraMode.ShadowCascades);
```

Comment:
was this a bug?

Reply:
This makes the debug view change. But nothing is showing just yet. When we have the proper debug view we will add it back in my opinion. 
So that is coming up next.